templates/sellix/get-blacklist/input.js

```
@@ -0,0 +1,28 @@
+/**
+ * ----------------------------------------------------------------------------------------------------
+ * Sellix GET Blacklist [Input]
+ *
+ * @author    Finn Lancaster
+ * @access    open
+ * @license   MIT
+ * @docs      https://dev-openapi.sellix.io/#operation/getBlacklists
+ *
+ * ----------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * Lets you select the input for your Node's run function
+ *
+ * @param {Params} params
+ * @param {Object} $trigger - This Flow's request object
+ * @param {Object} $nodes - Data from above Nodes
+ */
+ const nodeInput = ({ $trigger, $nodes }) => {
+  return {
+    method: "GET", // Required
+    url: "https://dev.sellix.io/v1/blacklists", // Required
```

It would be a good idea to pass the base url as an environment variable instead of being explicit in the input.

---

That makes sense, but I think it could get confusing to end-users, not to mention they then have to go through the docs to find the correct base url. I think a better idea would be to have an optional environment variable, which, if not set, defaults to the hardcoded one (current). What do you think about this?